configuration/config.go

```diff
@@ -16,36 +16,37 @@ type TupleArray []TupleItem
 
 // Config stores application configurations
 type Config struct {
-	Buildfile               string
-	Workdir                 string
-	NoCache                 bool
-	SuppressOutput          bool
-	RmTmpContainers         bool
-	ForceRmTmpContainer     bool
-	UniqueID                string
-	Logger                  logging.Logger
-	DockerHost              string
-	DockerCert              string
-	EnvVars                 TupleArray
-	BuildArgs               TupleArray
-	KeepSteps               bool
-	KeepArtifacts           bool
-	NoSquash                bool
-	NoPruneRmImages         bool
-	UseTLS                  bool
-	UseStatForPermissions	bool
-	FroceRmImages           bool
-	ApiPort                 int
-	ApiBinding              string
-	SecretService           bool
-	AllowAfterBuildCommands bool
-	SecretProviders         string
-	DockerMemory            string
-	DockerCPUSetCPUs        string
-	DockerCPUShares         int
-	UseAuthenticatedSecretServer bool
+	Buildfile                         string
+	Workdir                           string
+	NoCache                           bool
+	SuppressOutput                    bool
+	RmTmpContainers                   bool
+	ForceRmTmpContainer               bool
+	UniqueID                          string
+	Logger                            logging.Logger
+	DockerHost                        string
+	DockerCert                        string
+	EnvVars                           TupleArray
+	BuildArgs                         TupleArray
+	KeepSteps                         bool
+	KeepArtifacts                     bool
+	Network                           string
+	NoSquash                          bool
+	NoPruneRmImages                   bool
+	UseTLS                            bool
+	UseStatForPermissions             bool
+	FroceRmImages                     bool
+	ApiPort                           int
+	ApiBinding                        string
+	SecretService                     bool
+	AllowAfterBuildCommands           bool
+	SecretProviders                   string
+	DockerMemory                      string
+	DockerCPUSetCPUs                  string
+	DockerCPUShares                   int
+	UseAuthenticatedSecretServer      bool
```

Comment:
My IDEA - VS Code - seems to run `gofmt` automatically. Anyway, the sole change here is the addition of `Network                           string`.

Reply:
Oh, I'd like to run `gofmt` throughout the code-base, and also format checks on CI, so that we're freed from code formatting war 😆